src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java

```diff
@@ -180,7 +181,12 @@ private DefaultJudgmentSDOOrderForm getDefaultJudgmentFormTrial(CaseData caseDat
             .trialClinicalNegligenceAddSection(nonNull(caseData.getTrialClinicalNegligence()))
             .trialCreditHire(caseData.getTrialCreditHire())
             .trialCreditHireAddSection(nonNull(caseData.getTrialCreditHire()))
-            .trialHearingJudgesRecitalDJ(caseData.getTrialHearingJudgesRecitalDJ())
+            .sdoDJR2TrialCreditHireAddSection(nonNull(caseData.getSdoDJR2TrialCreditHire()))
+            .sdoDJR2TrialCreditHireDetailsAddSection(
+                (nonNull(caseData.getSdoDJR2TrialCreditHire())
+                    && nonNull(caseData.getSdoDJR2TrialCreditHire().getDetailsShowToggle())
+                    && caseData.getSdoDJR2TrialCreditHire().getDetailsShowToggle()
+                    .equals(List.of(AddOrRemoveToggle.ADD))))
```

Comment:
Should these attributes be added based on SdoR2Enabled flag. Is it ok to have to existing code. If it doesnt affect, then its fine.